### PR TITLE
fix some typo error

### PR DESCRIPTION
**Pull request by shuangzi52**
Task Description
correct some source file typos.

Solution Description
correct spell.

Passed Regressions
compile successfully

Upgrade Compatibility
No

Other Information
No

Release Note
No

#1543

### CODE REVIEW

File: src/sql/optimizer/ob_select_log_plan.cpp [hunk 1]
```diff
@@ -6736,7 +6736,7 @@ int ObSelectLogPlan::convert_project_columns(ObSelectStmt *stmt,
         expr->set_table_name(project_table_item->get_table_name());
         if (expr->is_virtual_generated_column()) {
           if (item->is_geo_ == true && expr->get_srs_id() != SPATIAL_COLUMN_SRID_MASK) {
-            // spatial index generated column, cannot projet from main table
+            // spatial index generated column, cannot project from main table
```
Comment (nautaa) [inline src/sql/optimizer/ob_select_log_plan.cpp:6739]: error: patch failed: src/sql/optimizer/ob_select_log_plan.cpp

please remove modification of this file.

**Review by nautaa — CHANGES_REQUESTED** [review]

File: src/sql/optimizer/ob_select_log_plan.cpp (hunk 1 above)
Comment (shuangzi52) [reply src/sql/optimizer/ob_select_log_plan.cpp]: I have revert the conflict modification, please review again, thank you.

**Comment by nautaa** [comment]
CONFLICT (content): Merge conflict in src/sql/engine/cmd/ob_partition_executor_utils.cpp.
please remove modification of this file.

**Comment by shuangzi52** [comment]
> CONFLICT (content): Merge conflict in src/sql/engine/cmd/ob_partition_executor_utils.cpp. please remove modification of this file.

fixed

**Pull request merged**